src/test-run/index.js

```diff
@@ -160,10 +167,22 @@ export default class TestRun extends AsyncEventEmitter {
                 onRequest:           hook.onRequest.bind(hook),
                 onConfigureResponse: hook._onConfigureResponse.bind(hook),
                 onResponse:          hook.onResponse.bind(hook)
-            });
+            }, err => this._onRequestHookMethodError(err, hook));
         });
     }
 
+    _onRequestHookMethodError (event, hook) {
+        let err = event.error;
+
+        if (err instanceof RequestHookNotImplementedMethodError === false) {
```

Comment:
Just proposal, maybe this way is more clear. 
```JS
const isRequestHookNotImplementedMethodError = err instanceof RequestHookNotImplementedMethodError

if (!isRequestHookNotImplementedMethodError ) {
...
}
```

Reply:
Ok. I will fix it.